stock_picking_auto_create_lot_qty/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Picking Auto Create Lot Quantity",
+    "summary": "Auto batch generation by quantity",
+    "version": "14.0.1.0.0",
+    "development_status": "Production/Stable",
+    "category": "stock",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "author": "Ooops, Cetmix OÜ, Odoo Community Association (OCA)",
```

Comment:
`Cetmix OÜ`?

Reply:
@pedrobaeza that's their legal name: https://www.odoocompanies.com/c-cetmix-o